src/tools/r2rdump/R2RSignature.cs

```diff
@@ -176,17 +196,17 @@ private string EmitMethodSignature(MethodSignature<string> methodSignature)
         }
 
         /// <summary>
-        /// Emit containing type and method name and extract the method signature from a method reference.
+        /// Emit containing type and member name.
         /// </summary>
-        /// <param name="methodRef">Method reference to format</param>
-        /// <param name="methodSignature">Output method signature</param>
-        private string EmitContainingTypeAndMethodName(MemberReference methodRef, string owningTypeOverride)
+        /// <param name="memberRef">Member reference to format</param>
+        /// <param name="owningTypeOverride">Optional override for the owning type, null = MemberReference.Parent</param>
+        private string EmitContainingTypeAndMemberName(MemberReference memberRef, string owningTypeOverride)
```

Comment:
I was in the middle of looking over the first commit and was going to ask you to refactor this method. But you beat me to it :)

Reply:
For some reason, I'm often unable to spot such minor lapses in my changes before I actually publish them. Perhaps it's a form of exhibitionism or something ;-).